server/fusor_server.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "net-ssh", '2.9.2' # last ruby 1.9 version
   s.add_dependency 'net-scp'
   s.add_dependency "sys-filesystem"
+  s.add_dependency "kaminari"
```

Comment:
ember-infinity works out of the box with kaminar. It would need to modification if it needs to work with will_paginate instead.